collections/_article/dev-snapshot-godot-4-5-dev-2.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Dev snapshot: Godot 4.5 dev 2"
+excerpt: This is where the fun begins
+categories: [pre-release]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-5-dev-2.webp
+image_caption_title: Fortune Avenue
+image_caption_description: A game by Binogure Studio
+date: 2025-04-08 12:00:00
+---
+
+With the [4.4 release](/releases/4.4/) a little over a month ago, one might've expected content updates to trickle out slowly for our initial 4.5 development snapshots. …Well, maybe not after seeing the [4.5 dev 1](/article/dev-snapshot-godot-4-5-dev-1/) snapshot, but **surely** they couldn't follow that up with another flood of anticipated changes, right?
+
+You fools. You underestimate the passion of our community yet again, as **250** improvements are ready to roll for this snapshot. What's more is that, unlike last time, we're not *just* in bugfix territory any more. Now the time for proper enhancements and features entirely unique to this development cycle has come, and this blogpost will aim to highlight them to the best of our ability. As always: with new features comes new bugs (probably), so the sooner we can get feedback and bug reports in, the better.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.5.dev2/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+---
+
+*The cover illustration is from* [**Fortune Avenue**](https://store.steampowered.com/app/1810050/Fortune_Avenue/?curator_clanid=41324400), *a capitalism simulator where you shrewdly extort and outmaneuver your friends in a chaotic, board-game environment. It is developed by [Binogure Studio](https://www.binogure-studio.com/). You can wishlist the game [on Steam](https://store.steampowered.com/app/1810050/Fortune_Avenue/?curator_clanid=41324400) and follow the developers on [Bluesky](https://bsky.app/profile/binogure-studio.com).*
+
+## Highlights
+
+In case you missed them, see the [4.5 dev 1](/article/dev-snapshot-godot-4-5-dev-1/) release notes for an overview of some key features which was already in that snapshot, and are therefore still available for testing in dev 2.
```

Comment:
```suggestion
In case you missed them, see the [4.5 dev 1](/article/dev-snapshot-godot-4-5-dev-1/) release notes for an overview of some key features which were already in that snapshot, and are therefore still available for testing in dev 2.
```